### ISSUE TITLE
mod requests

### ISSUE THREAD

**Issue by odenthefish** [AlexTheGre4t/MCTNGIssues#14]
Garden stuff 
adds things like flowerpots and window boxes
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2163513-garden-stuff-v-1-5-3-updated-mar-19-15

mariculture
adds things like diving suits, speed boats and fish breeding ( like forestry bees ) Enchiridion is required to use the books ( I believe that it only adds a few book related things)
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1292346-mariculture-1-2-4-1-updated-13-03-2015

modular systems
adds modular furnaces, ore processing, storage and enchanting
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1291862-modular-systems-formally-modular-furnace

**Comment by ryuben** [comment]
https://docs.google.com/spreadsheets/d/1uUsBctyUNs4qOoVi-Yp1CSMiIsHQFOseQ-ijryjiuZE/edit#gid=1362999793

**Comment by ryuben** [comment]
https://github.com/Ephys/TinkersSteelworks

plx

**Comment by ryuben** [comment]
^ Yes or no?

**Comment by Xariyl** [comment]
My votes:
Mariculture : maybe/Yes.

Gardent Stuff: Yes.

Modular Systems: maybe/no. 

Tinkers Steelworks: Yes. 

---

Garden Stuff is a good decorative mod that plays well with existing mods in the pack. There are no significant gameplay changes. I think this would be a good addition.

Mariculture, the last time I played it, added so much world gen that it lagged my single player world so much it was unplayable. It adds a lot of neat features and has a lot of compatibility with existing mods in the pack. I would conservatively accept it but evaluation needs to be done with all the things it adds. 

Modular Systems, while an interesting gameplay mechanic, seems largely unnecessary and I'm concerned it may add a lot of overlap with existing mods. I've not used this mod before, so my first instinct is No. Needs evaluation.

Tinkers Steelworks adds a lot that looks interesting. I've not used it before, but I see nothing against it. I have no concerns with adding this to the modpack.

**Comment by ryuben** [comment]
so we can add garden stuff and tinkers ? 

when does alex get back?

**Comment by Xariyl** [comment]
I know he's pretty busy. I'm looking forward to BiblioCraft and Flood Lights too, hopefully they can be added to the next pack release.

**Comment by AlexTheGre4t** [comment]
I'm definitely gonna have a look at these. I'm going away until Sunday. I'll resume work when I get back.

**Comment by ryuben** [comment]
i didn't approve any time off - did you Xariyl?

**Comment by Xariyl** [comment]
Yeah, I did. He's distracted so I can MineTweak turning potatoes into ender pearls, he'll accept the pull request without reading it :)

**Comment by AlexTheGre4t** [comment]
Heh, I like reading your pull requests Xariyl, it's kinda nice seeing some scripts that aren't yours for once :D

I approved time off way back in May when I got pack permissions with the Toad himself... :P

**Comment by odenthefish** [comment]
i have no clue on how to quote you xariyl but i didn't see any major lag caused by mariculture although i did get some pretty heavy tickrate lag when generating new chunks but it always went back to normal 
(but all bets are off when my funky locomotion quarrys are running eek) and the generation can be disabled i believe (maybe even where you can select certain things to disable)
i do agree on modular systems being kind of overkill because as you said most of it is already installed in one way or another
i don't think i have ever played with tinkers steelworks but from what i've seen it has a pretty cool gameplay mechanic

**Comment by ryuben** [comment]
i am hesitant to say this but i agree with the fish
